lib/src/modules/common/file_upload/file_uploader.dart

```diff
@@ -1,44 +1,68 @@
-import 'package:fa_flutter_core/fa_flutter_core.dart';
+import 'package:image_picker/image_picker.dart';
 import 'package:file_picker/file_picker.dart';
-import 'package:open_file/src/platform/open_file.dart';
+import 'package:open_file/open_file.dart';
 
 class FileService {
   FileService._();
 
-  /// Return the Clicked Image
+  /// Picks an image and returns a `PlatformFile`.
+  /// The file size should be in MB.
+  /// No limit for file size if [requiredFileSize] is null
+
   static Future<PlatformFile?> pickImage(
```

Comment:
mention it that this function thows error when picked file size exceeds requiredFileSize.